pkg/controller/clusterpool/clusterpool_controller_test.go

```diff
@@ -519,3 +523,755 @@ func TestReconcileClusterPool(t *testing.T) {
 		})
 	}
 }
+
+func TestReconcileRBAC(t *testing.T) {
+	scheme := runtime.NewScheme()
+	hivev1.AddToScheme(scheme)
+	corev1.AddToScheme(scheme)
+	rbacv1.AddToScheme(scheme)
+
+	tests := []struct {
+		name string
+
+		existing []runtime.Object
+
+		expectedBindings []rbacv1.RoleBinding
+		expectedErr      string
+	}{{
+		name:     "no binding referring to cluster role",
+		existing: []runtime.Object{},
+	}, {
+		name: "no binding referring to cluster role",
```

Comment:
name is duplicated with the test above.

Reply:
the name is same because technically it's testing the same thing but just a different permutation really.. 
```console
go test -v ./pkg/controller/clusterpool/... -run "TestReconcileRBAC/no_binding_referring_to_cluster_role"
=== RUN   TestReconcileRBAC
=== RUN   TestReconcileRBAC/no_binding_referring_to_cluster_role
=== RUN   TestReconcileRBAC/no_binding_referring_to_cluster_role#01
--- PASS: TestReconcileRBAC (0.00s)
    --- PASS: TestReconcileRBAC/no_binding_referring_to_cluster_role (0.00s)
    --- PASS: TestReconcileRBAC/no_binding_referring_to_cluster_role#01 (0.00s)
PASS
ok      github.com/openshift/hive/pkg/controller/clusterpool    0.031s
```
and go test automatically adds number to the same test name.

should i add one explicitly here?

Reply:
Peculiar, we don't do this anywhere today afaik and typically try to give the tests unique names, this could get a little funky for anyone looking to just go run -test for a specific test, but probably not a huge deal. I think my vote would still be to give them unique names that cover the intent though.